docs/doc/14-sql-commands/10-dml/dml-copy-into-table.md

```diff
@@ -221,81 +214,103 @@ copyOptions ::=
 When importing large volumes of data, such as logs, it is recommended to set both `PURGE` and `FORCE` to True. This ensures efficient data import without the need for interaction with the Meta server (updating the copied-files set). However, it is important to be aware that this may lead to duplicate data imports.
 :::
 
-### Output
+## Output
 
 COPY INTO provides a summary of the data loading results with these columns:
 
-| Column           | DataType | Nullable | Description                                |
+| Column           | Type     | Nullable | Description                                |
 |------------------|----------|----------|--------------------------------------------|
 | FILE             | VARCHAR  | NO       | The relative path to the source file.       |
 | ROWS_LOADED      | INT      | NO       | The number of rows loaded from the source file. |
 | ERRORS_SEEN      | INT      | NO       | Number of error rows in the source file    |
 | FIRST_ERROR      | VARCHAR  | YES      | The first error found in the source file.             |
 | FIRST_ERROR_LINE | INT      | YES      | Line number of the first error.             |
 
+## Managing Parallel Processing
 
-## Examples
+In Databend, the `max_threads` setting specifies the maximum number of threads that can be utilized to execute a request. By default, this value is typically set to match the number of CPU cores available on the machine.
```

Comment:
It's better not let the user know the settings like `max_threads`, all is done internal for them.